packages/protocol/lib/bytecode.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
biggest nit ever, I'm almost embarrassed, but all the other files in this directory end in `-utils` wondering if `bytecode-utils` will make it more obvious this is a file of helper functions.

Reply:
Always appreciate a good nit! I did do this deliberately because I've never been a huge fan of the `-utils` suffix going on everything in `lib/`, but could change it for consistency.